docs/guides/nrfconnect_factory_data_configuration.md

```diff
@@ -30,29 +30,30 @@ data secure by applying hardware write protection.
 
 <hr>
 
--   [Configuring factory data for the nRF Connect examples](#configuring-factory-data-for-the-nrf-connect-examples)
-    -   [Overview](#overview)
-        -   [Factory data component table](#factory-data-component-table)
-        -   [Factory data format](#factory-data-format)
-            -   [Appearance field description](#appearance-field-description)
-    -   [Enabling factory data support](#enabling-factory-data-support)
-    -   [Generating factory data](#generating-factory-data)
-        -   [Creating the factory data JSON file with the first script](#creating-the-factory-data-json-file-with-the-first-script)
-        -   [How to set user data](#how-to-set-user-data)
-            -   [How to handle user data](#how-to-handle-user-data)
-        -   [Verifying using the JSON Schema tool](#verifying-using-the-json-schema-tool)
-            -   [Option 1: Using the php-json-schema tool](#option-1-using-the-php-json-schema-tool)
-            -   [Option 2: Using a website validator](#option-2-using-a-website-validator)
-            -   [Option 3: Using the nRF Connect Python script](#option-3-using-the-nrf-connect-python-script)
-        -   [Generating onboarding codes](#generating-onboarding-codes)
-            -   [Enabling onboarding codes generation within the build system](#enabling-onboarding-codes-generation-within-the-build-system)
-        -   [Preparing factory data partition on a device](#preparing-factory-data-partition-on-a-device)
-        -   [Creating a factory data partition with the second script](#creating-a-factory-data-partition-with-the-second-script)
-    -   [Building an example with factory data](#building-an-example-with-factory-data)
-        -   [Providing factory data parameters as a build argument list](#providing-factory-data-parameters-as-a-build-argument-list)
-        -   [Setting factory data parameters using interactive Kconfig interfaces](#setting-factory-data-parameters-using-interactive-kconfig-interfaces)
-    -   [Programming factory data](#programming-factory-data)
-    -   [Using own factory data implementation](#using-own-factory-data-implementation)
+- [Configuring factory data for the nRF Connect examples](#configuring-factory-data-for-the-nrf-connect-examples)
```

Comment:
This seems like it's only white space changes, correct?

Reply:
White space changed and one line was added regarding the new topic. But I guess the white space will change when I open a PR in Matter upstream because there is a different code-formatting tool used.